Tooling-Landscape/Unanimous-Understanding/OSS-Tooling-Landscape-Glossary.md

```diff
@@ -3,46 +3,66 @@
 ## Company External
 
 ### Source Code Repository
-Systems or services that provide source code. These systems or services can provide plain source code only or – in addition – also have version control. 
+Systems or services that provide source code. This is typically a version control system. The compliance tooling obtains the source code of 3rd party dependencies, i.e., FOSS components from such repositories
 
-### Build Artefacts Repository
-Systems or services that provides (binary) software artefacts and metadata stored in a defined directory structure which is used to retrieve artefacts during a build process. Software artefacts may be of type proprietary, commercial, FOSS, …
+### (Build) Artifact Repository
+Systems or services that provides (binary) software artifacts and metadata stored in a defined directory structure which is used to retrieve artifacts during a build process. Software artifacts may be of type proprietary, commercial, FOSS. The Artifact repository may also be used to store the source code of FOSS components besides the binary artifact for usage within the compliance process, e.g., for starting a license scan or creating the source code bundle to be delivered with the product.
 
-### Security Vulnerabilities
+### Security Vulnerability Database
 Systems or services that provide information about publicly known security vulnerabilities of software artefacts.
 
-### License Obligations
-Systems or services that provide (machine readable?) Data about obligations and prohibitions of (FOSS) licenses.
+### Public License Master Database
+Publicly available information about Open Source licenses (e.g. commonly accepted license identifiers). It could furthermore provide (potentially machine readable) data about the corresponding obligations and prohibitions of a license.
 
-### Public License Master Data
-Publicly available information about Open Source licenses (e.g. commonly accepted license identifiers). 
-
-### FOSS Meta Data
+### FOSS Metadatabase
```

Comment:
term "metadatabase" seems to be odd for the intended purpose, searching the Internet yields for example: https://en.wikipedia.org/wiki/Metadatabase